models/issue.go

```diff
@@ -974,13 +980,8 @@ type IssueUser struct {
 	ID          int64 `xorm:"pk autoincr"`
 	UID         int64 `xorm:"INDEX"` // User ID.
 	IssueID     int64
-	RepoID      int64 `xorm:"INDEX"`
-	MilestoneID int64
 	IsRead      bool
-	IsAssigned  bool
 	IsMentioned bool
```

Comment:
The `issue` table has an `assignee_id` column, and gitea only supports one assignee per repo